docs/sdk_docs/docs/KServeClient.md

```diff
@@ -0,0 +1,353 @@
+
+### Example
+
+```python
+from kserve import KServeClient
+
+kserve = KServeClient()
+kserve.get('flower-sample', namespace='kubeflow')
+```
+The API also support watching the specified InferenceService or all InferenceService in the namespace.
+```python
+from kserve import KServeClient
+
+kserve = KServeClient()
+kserve.get('flower-sample', namespace='kubeflow', watch=True, timeout_seconds=120)
+```
+The outputs will be as following. Stop watching if InferenceService reaches the optional specified `timeout_seconds` or once the InferenceService overall status `READY` is `True`.
+```sh
+NAME                 READY      DEFAULT_TRAFFIC CANARY_TRAFFIC  URL                                               
+flower-sample        Unknown                                    http://flower-sample.kubeflow.example.com         
+flower-sample        Unknown    90               10             http://flower-sample.kubeflow.example.com         
+flower-sample        True       90               10             http://flower-sample.kubeflow.example.com         
+```
+
+
+### Parameters
+Name | Type |  Description | Notes
+------------ | ------------- | ------------- | -------------
+name  | str | InferenceService name. If the `name` is not specified, it will get or watch all InferenceServices in the namespace.| Optional. |
+namespace | str | The InferenceService's namespace. Defaults to current or default namespace.| Optional |
+watch | bool | Watch the specified InferenceService or all InferenceService in the namespace if `True`, otherwise will return object for the specified InferenceService or all InferenceService in the namespace. Stop watching if InferenceService reaches the optional specified `timeout_seconds` or once the speficed InferenceService overall status `READY` is `True` (Only if the `name` is speficed). | Optional |
+timeout_seconds | int | Timeout seconds for watching. Defaults to 600. | Optional |
+
+### Return type
+object
+
+
+## patch
+> patch(name, inferenceservice, namespace=None, watch=False, timeout_seconds=600)
+
+Patch the created InferenceService in the specified namespace.
+
+Note that if you want to set the field from existing value to `None`, `patch` API may not work, you need to use [replace](#replace) API to remove the field value.
+
+### Example
+
+```python
+from kubernetes import client
+from kserve import constants
+from kserve import V1beta1EndpointSpec
+from kserve import V1beta1PredictorSpec
+from kserve import V1alpha2TensorflowSpec
+from kserve import V1beta1InferenceServiceSpec
+from kserve import V1beta1InferenceService
+from kserve import KServeClient
+
+default_model_spec = V1beta1EndpointSpec(predictor=V1beta1PredictorSpec(tensorflow=V1alpha2TensorflowSpec(
+    storage_uri='gs://kfserving-samples/models/tensorflow/flowers')))
+canary_model_spec = V1beta1EndpointSpec(predictor=V1beta1PredictorSpec(tensorflow=V1alpha2TensorflowSpec(
+    storage_uri='gs://kfserving-samples/models/tensorflow/flowers')))
```

Comment:
the canary API is changed in v1beta1, please help checkout e2e test example https://github.com/kserve/kserve/blob/master/test/e2e/predictor/test_canary.py#L32